src/structured-data/structured-data-tags.js

```diff
@@ -178,6 +178,7 @@ function generateWebSiteData(structuredData = {}, story = {}, publisherConfig =
 export function StructuredDataTags({structuredData = {}}, config, pageType, response = {}, {url}) {
   const tags = [];
   const {story = {}} = response.data || {};
+  const entities = get (response, ["data", "linkedEntities"], null);
```

Comment:
give default as `[]` and check for length

Reply:
`get (response, ["data", "linkedEntities"], null) || [];` cos lodash return null if the value is null instead of using fallback.